src/_zkapauthorizer/_plugin.py

```diff
@@ -17,16 +17,26 @@
 Tahoe-LAFS.
 """
 
+from __future__ import absolute_import
+
 import random
 from datetime import datetime
 from functools import partial
 from weakref import WeakValueDictionary
 
+try:
+    from typing import Callable
```

Comment:
Are any of these types checked, or are the aspirational/used for IDE annotations only at the moment?

Reply:
aspirational